pkg/config/msp_test.go

```diff
@@ -0,0 +1,628 @@
+										"type": 3,
+										"value": {
+											"rule": "MAJORITY",
+											"sub_policy": "Endorsement"
+										}
+									},
+									"version": "0"
+								},
+								"LifecycleEndorsement": {
+									"mod_policy": "Admins",
+									"policy": {
+										"type": 3,
+										"value": {
+											"rule": "MAJORITY",
+											"sub_policy": "Endorsement"
+										}
+									},
+									"version": "0"
+								},
+								"Readers": {
+									"mod_policy": "Admins",
+									"policy": {
+										"type": 3,
+										"value": {
+											"rule": "ANY",
+											"sub_policy": "Readers"
+										}
+									},
+									"version": "0"
+								},
+								"Writers": {
+									"mod_policy": "Admins",
+									"policy": {
+										"type": 3,
+										"value": {
+											"rule": "ANY",
+											"sub_policy": "Writers"
+										}
+									},
+									"version": "0"
+								}
+							},
+							"values": {
+								"AnchorPeers": {
+									"mod_policy": "Admins",
+									"value": {
+										"anchor_peers": [
+											{
+												"host": "host2",
+												"port": 123
+											}
+										]
+									},
+									"version": "0"
+								},
+								"MSP": {
+									"mod_policy": "Admins",
+									"value": {
+										"config": null,
+										"type": 0
+									},
+									"version": "0"
+								}
+							},
+							"version": "0"
+						}
+					},
+					"mod_policy": "Admins",
+					"policies": {
+						"Admins": {
+							"mod_policy": "Admins",
+							"policy": {
+								"type": 3,
+								"value": {
+									"rule": "MAJORITY",
+									"sub_policy": "Admins"
+								}
+							},
+							"version": "0"
+						},
+						"Readers": {
+							"mod_policy": "Admins",
+							"policy": {
+								"type": 3,
+								"value": {
+									"rule": "ANY",
+									"sub_policy": "Readers"
+								}
+							},
+							"version": "0"
+						},
+						"Writers": {
+							"mod_policy": "Admins",
+							"policy": {
+								"type": 3,
+								"value": {
+									"rule": "ANY",
+									"sub_policy": "Writers"
+								}
+							},
+							"version": "0"
+						}
+					},
+					"values": {
+						"ACLs": {
+							"mod_policy": "Admins",
+							"value": {
+								"acls": {
+									"acl1": {
+										"policy_ref": "hi"
+									}
+								}
+							},
+							"version": "0"
+						},
+						"Capabilities": {
+							"mod_policy": "Admins",
+							"value": {
+								"capabilities": {
+									"V1_3": {}
+								}
+							},
+							"version": "0"
+						}
+					},
+					"version": "0"
+				}
+			},
+			"mod_policy": "",
+			"policies": {},
+			"values": {},
+			"version": "0"
+		},
+		"sequence": "0"
+	}
+	`
+
+	expectedConfigProto := &cb.Config{}
+	err = protolator.DeepUnmarshalJSON(bytes.NewBufferString(expectedConfig), expectedConfigProto)
+	gt.Expect(err).NotTo(HaveOccurred())
+
+	gt.Expect(proto.Equal(config, expectedConfigProto)).To(BeTrue())
```

Comment:
nit: this is fine, but if it ever does fail it will just show as `Expected false to be true`, the values themselves should be equivalent as well so `gt.Expect(config).To(Equal(expectedConfigProto))` should be fine here and will actually print out a diff if it's not equal

Reply:
Done